scripts/setup/setup-certbot

```diff
@@ -112,7 +117,7 @@ symlink_with_backup() {
 if [ ${#deploy_hook} -eq 0 ]; then
     # If no deploy hook was specified, assume we're deploying to the default
     # location Zulip wants.
-    CERT_DIR=/etc/letsencrypt/live/"$DOMAIN"
+    CERT_DIR=/etc/letsencrypt/live/"${DOMAIN}"
```

Comment:
That makes no difference.

Reply:
Sorry, this looks like an artifact of some changes I made, will fix.